### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by gmethvin**

### CODE REVIEW

**Comment by mkurz** [comment]
Because you upgrade Twirl in this PR: Refering [this comment](https://github.com/playframework/twirl/pull/100#issuecomment-217322197), shouldn't we add 
```
TwirlKeys.constructorAnnotations += "@javax.inject.Inject()"
```
?

**Comment by gmethvin** [comment]
@mkurz Sure, good idea.

File: framework/project/Dependencies.scala [hunk 1]
```diff
@@ -108,11 +108,11 @@ object Dependencies {
...
   ).map(_ % Test)
 
-  val jodatime = "joda-time" % "joda-time" % "2.9.3"
+  val jodatime = "joda-time" % "joda-time" % "2.9.4"
   val jodaConvert = "org.joda" % "joda-convert" % "1.8.1"
 
   val guiceDeps = Seq(
-    "com.google.inject" % "guice" % "4.0",
+    "com.google.inject" % "guice" % "4.1.0",
     "com.google.inject.extensions" % "guice-assistedinject" % "4.0"
```
Comment (schmitch) [inline framework/project/Dependencies.scala]: any reason why this isn't updated aswell?

**Comment by gmethvin** [comment]
Looking into if the build failure is related to this.

**Pull request closed without merging**